build/scripts/build_digests_bundle.sh

```diff
@@ -20,6 +24,7 @@ Arguments:
   --push <IMAGE>          : Push processed digests bundle to <IMAGE> (required). Pushing the bundle image to a
                             repository is required because opm render works from a remote repository.
   --container-tool <TOOL> : Use specific container tool for building/pushing images (default: use podman).
+  --multi-arch            : Create images for the following architectures: $ARCHITECTURES
```

Comment:
Maybe add a note here that this will use docker buildx, ignoring settings from `--container-tool`